src/content/docs/codestream/start-here/new-relic-codestream-quick-start.mdx

```diff
@@ -0,0 +1,22 @@
+---
+title: New Relic and CodeStream quick start
+metaDescription: "The quickest way to get started with CodeStream and New Relic."
+---
+
+Follow these quick steps to get started with New Relic and CodeStream.
+
+## Start with a New Relic account [#account]
+
+If you don't have a New Relic account, sign up at [newrelic.com/signup.](https://newrelic.com/signup). It's free, forever!
+
+## Install CodeStream via New Relic [#quickstart]
+
+Use our [CodeStream quickstart](ADD URL) to install it for your IDE.
+
+### Install CodeStream in your IDE [#install-ide]
+
+You can also install CodeStream via your VS Code and Visual Studio extensions menu or Jetbrains plugins menu.
+
+## Install New Relic [#install-new-relic]
+
+Follow the steps in our [Add your data](https://one.newrelic.com/launcher/nr1-core.settings?pane=eyJuZXJkbGV0SWQiOiJ0dWNzb24ucGxnLWluc3RydW1lbnQtZXZlcnl0aGluZyJ9) UI page to get data flowing in. For your first install, we recommend the **Guided install** option, which will setup many integrations with a single command.
```

Comment:
We are also going to have an ability for customers to instrument their code right from the IDE.  As of right now this is giong to be just for Java, Node.js and .NET Core (.NET Framework might not be done in time)

Reply:
Is this even worth mentioning in this quick start doc? It's such an edge case at this point. I'd prefer to mention it in the more detailed CodeStream and New Relic doc.